pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -152,6 +222,44 @@ var (
 		Obj()
 )
 
+func reserve(claim *resourcev1alpha2.ResourceClaim, pod *v1.Pod) *resourcev1alpha2.ResourceClaim {
+	return st.FromResourceClaim(claim).
+		ReservedFor(resourcev1alpha2.ResourceClaimConsumerReference{Resource: "pods", Name: pod.Name, UID: types.UID(pod.UID)}).
```

Comment:
Maybe you can have a `ReservedForPod` method?

Reply:
Added.